components/notebook-controller/Dockerfile

```diff
@@ -57,6 +57,6 @@ COPY --from=builder /workspace/notebook-controller/bin/manager /manager
 COPY --from=builder /workspace/notebook-controller/third_party/license.txt third_party/license.txt
 
 ## Switch to a non-root user
-USER rhods
+USER 1001:0
```

Comment:
OpenShift default is to set random uid and 0 gid, https://access.redhat.com/documentation/cn/openshift_container_platform/4.15/html/images/creating-images#use-uid_create-images

I feel like we should keep close to that

Reply:
Well, I read this link before, since you have it in the description... so I probably missed something here.
My point is - why setting also GID here as it is 0 by default anyway. From your answer, I understand that it's just a precaution, correct?

Reply:
Exactly.